test/params.js

```diff
@@ -89,44 +89,44 @@ describe('LDNODE params', function () {
     });
   });
 
```

Comment:
since it can't be tested in that way.
In other words, now mount is a parameter of createServer, while this is a test for createApp, so mount can't be specified